tests/lib/rules/strict-component-boundaries.js

```diff
@@ -25,6 +25,14 @@ ruleTester.run('strict-component-boundaries', rule, {
       code: `import {someThing} from '../OtherComponent';`,
       parserOptions,
     },
+    {
+      code: `import fs from 'fs';`,
```

Comment:
these 2 added tests are not right, they should have something that would otherwise trigger this rule, but because they are in node modules (or a core package) they would be valid.